ci/release-image/Dockerfile

```diff
@@ -1,20 +1,28 @@
 FROM debian:10
 
-RUN apt-get update
-RUN apt-get install -y curl
+RUN apt-get update \
+ && apt-get install -y \
+    curl \
+    dumb-init \
+    htop \
+    locales \
+    man \
+    nano \
+    git \
+    procps \
+    ssh \
+    sudo \
+    vim \
+  && rm -rf /var/lib/apt/lists/*
 
 # https://wiki.debian.org/Locale#Manually
-RUN apt-get install -y locales
-RUN sed -i "s/# en_US.UTF-8/en_US.UTF-8/" /etc/locale.gen
-RUN locale-gen
+RUN sed -i "s/# en_US.UTF-8/en_US.UTF-8/" /etc/locale.gen \
+  && locale-gen
 ENV LANG=en_US.UTF-8
 
 RUN chsh -s /bin/bash
 ENV SHELL=/bin/bash
 
-RUN apt-get install -y dumb-init sudo
-RUN apt-get install -y man procps vim nano htop ssh git
```

Comment:
For example having these two installs right after each other does not improve anything and just adds another layer which decreases performance.